src/app/layout/navbar/navbar.component.html

```diff
@@ -1,5 +1,12 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <div class="container">
-    <a class="navbar-brand" href="#">GP4Tech</a>
+<nav class="navbar-fixed">
+  <div class="nav-wrapper">
+    <div class="container">
+      <a href="#" class="brand-logo left">GP4Tech</a>
+      <ul id="nav-mobile" class="left hide-on-med-and-down links">
```

Comment:
I'm not sure, but I think that we are not using that id for anything. Also, this nav isn't the mobile one.